Blueprints/BlueprintDefinitions/vs2022/MessageProcessingFramework/template/src/BlueprintBaseName.1/Readme.md

```diff
@@ -0,0 +1,66 @@
+# AWS Message Processing Framework for .NET Template
+
+This starter project consists of:
+* Functions.cs - class file containing a class with a two Lambda function handlers, one for sending mesages, one for processing them
+* Startup.cs - default argument settings for use with Visual Studio and command line deployment tools for AWS
+* GreetingMessage.cs* - Represents a single message
+* serverless.template - A CloudFormation template to deploy both functions. It also creates a new SQS queue that will the functions will send to and recieve messages from. 
```

Comment:
The `serverless.template` is missing in the PR.

Reply:
Oops, added in [4ad2937](https://github.com/aws/aws-lambda-dotnet/pull/1707/commits/4ad293784c1044b65deecb54b426b6520891b39a)